clang/test/CodeGenCXX/matrix-type.cpp

```diff
@@ -368,3 +368,36 @@ void test_use_matrix_2() {
 
   selector<2> r5 = use_matrix_3(m1);
 }
+
+// CHECK-LABEL: define void @_Z22test_pseudo_destructorv()
+// CHECK-NEXT: entry:
+// CHECK-NEXT:   %a = alloca [25 x double], align 8
```

Comment:
> I see the rest of the test doesn't

This is generally my excuse for not using `[[]]` because it’s a bit tedious to do that if there are a lot of tests ;Þ